### ISSUE TITLE
Detecting wireless for nodes/hubs all on local machine

### ISSUE THREAD

**Issue by stonier** [robotics-in-concert/rocon_multimaster#232]
First thing is this is giving the wrong impression of the system.

Secondly, maybe needs a third state, LOCAL, WIRED, WIRELESS. Dont know if this is possible.

**Comment by piyushk** [comment]
We should be able to check if the hub IP is a local IP address through the network_interface_manager wrapper. I'll test some code and get back to you.

**Comment by stonier** [comment]
I just had to do something similar for conductor/app manager checks when flagging them to deal with only local clients/controllers.

https://github.com/robotics-in-concert/rocon_concert/issues/147

Rosgraph has some easy [network api](https://github.com/ros/ros_comm/blob/hydro-devel/tools/rosgraph/src/rosgraph/network.py#L164). I wonder if this can be used here too.

**Comment by piyushk** [comment]
You're right. The rosgraph API is using the same code I was planning on using, and it has probably gone through a fair amount of testing over the years.

One of the mistakes in the code is that network information is being detected independent of the actual hub we connect to. We should probably fix https://github.com/robotics-in-concert/rocon_multimaster/issues/223 the same time as we fix this error.

**Comment by stonier** [comment]
ok.
